server/routes/activitiesMigration/activitiesMigrationController.ts

```diff
@@ -42,8 +42,26 @@ export default class ActivitiesMigrationController {
 
     const decoratedMigrations = migrations.map(ActivitiesMigrationController.withFilter).map(history => ({
       ...history,
-      applicationInsightsLink: ActivitiesMigrationController.applicationInsightsUrl(
-        ActivitiesMigrationController.alreadyMigratedApplicationInsightsQuery(history.whenStarted, history.whenEnded),
+      appInsightsAlreadyMigratedLink: ActivitiesMigrationController.applicationInsightsUrl(
+        ActivitiesMigrationController.alreadyMigratedAppInsightsQuery(
+          history.migrationId,
+          history.whenStarted,
+          history.whenEnded,
+        ),
+      ),
+      appInsightsFailuresLink: ActivitiesMigrationController.applicationInsightsUrlNoTimespan(
+        ActivitiesMigrationController.failedMigrationAppInsightsQuery(
+          history.migrationId,
+          history.whenStarted,
+          history.whenEnded,
+        ),
+      ),
+      appInsightsFullMigrationLink: ActivitiesMigrationController.applicationInsightsUrlNoTimespan(
+        ActivitiesMigrationController.fullMigrationAppInsightsQuery(
+          history.migrationId,
+          history.whenStarted,
+          history.whenEnded,
+        ),
```

Comment:
This is the main change:

- dropping the existing failures page (not helpful to our users) and replacing with a link to an App Insights query that shows failures and their reasons
- replacing the already migrated App Insights query with a better one
- adding a link to show all App Insights events generated during the migration